tests/integrations/aws_lambda/test_aws.py

```diff
@@ -183,27 +179,23 @@ def inner(
         response["Payload"] = json.loads(response["Payload"].read().decode("utf-8"))
         del response["ResponseMetadata"]
 
-        events = []
-        envelopes = []
+        envelope_item = []
```

Comment:
Why not keep calling this `envelopes`? That's the usual convention in our tests. Or maybe at least `envelope_items`, since it's a collection of items rather than one item?

Reply:
I made the change because we are actually capturing envelope `Item`s here, not `Envelope`s. The previous naming confused me when I was looking at the tests, because I thought these were `Envelope` objects, and I thought I would have to somehow manually extract the events (i.e. `Item`s) from them. The name `envelope_items` makes it clear that we are dealing with `Item` objects, not `Envelope` objects, here.

I believe the that in the "usual convention" you are referring to, we are capturing the envelopes using the `capture_envelopes` fixture, which returns a `list[Envelope]`. Therefore, while the `envelopes` name is the appropriate choice in those cases, the distinct `envelope_items` name here is warranted because here we have a `list[Item]`.

You are right though, we should be calling this variable `envelope_items` rather than `envelope_item`. I will make this change.

Reply:
Then let's go with `envelope_items` 👍🏻